guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,321 @@
+* For more information, see https://access.redhat.com/solutions/7030156[How to in-place upgrade an offline / disconnected RHEL 8 machine to RHEL 9 with Leapp?]
+endif::[]
+
+ifdef::satellite[]
+[NOTE]
+====
+* {Project} supports DEFAULT and FIPS crypto-policies.
+The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+* In-place upgrade of {EL} systems in FIPS mode is not supported by {Team}.
+Turning FIPS off, upgrading from {EL} 8 to {EL} 9, and then turning FIPS on is not supported either.
+Instead, migrate your {Project} {ProductVersion} to a freshly installed {EL} 8 system with FIPS mode enabled.
+For more information, see xref:migrating-{project-context}-to-a-new-el-system_{context}[]. <<<<<check link>>>>>
+====
+endif::[]
+
+.Procedure
+. Configure the repositories to obtain Leapp.
+ifdef::foreman-el,katello[]
++
+On CentOS, configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/[@theforeman/leapp COPR Repository], which contains newer Leapp packages than those shipped by https://wiki.almalinux.org/elevate/[AlmaLinux/ELevate], and support {Project} or {SmartProxy} upgrades:
++
+----
+# curl -o /etc/yum.repos.d/theforeman-leapp.repo https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/repo/epel-7/group_theforeman-leapp-epel-7.repo
+----
+endif::[]
++
+On {RHEL}, enable the `rhel-9-server-extras-rpms` repository:
++
+----
+# subscription-manager repos --enable=rhel-9-server-extras-rpms
+----
+
+. Install required packages:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
++
+----
+# {package-install-project} leapp leapp-repository
+----
+ifdef::satellite[]
+
+. Set up the following repositories to perform the upgrade in a disconnected environment:
+.. `/etc/yum.repos.d/rhel8.repo`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[BaseOS]
+name={RepoRHEL8BaseOS}
+baseurl=http://_server.example.com_/rhel9/BaseOS/
+
+[AppStream]
+name={RepoRHEL8AppStream}
+baseurl=http://_server.example.com_/rhel9/AppStream/
+----
+.. `/etc/yum.repos.d/{project-context}.repo:`
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={RepoRHEL8ServerSatelliteServerProductVersion}
+baseurl=http://_server.example.com_/sat6/Satellite/
+
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+baseurl=http://_server.example.com_/sat6/Maintenance/
+----
+endif::[]
+
+ifdef::foreman-el,katello[]
+. Install additional OS specific packages (`leapp-data-almalinux` for AlmaLinux, `leapp-data-centos` for CentOS Stream, or `leapp-data-rocky` for Rocky Linux).
+Note that this is not required for {RHEL} based installations.
++
+----
+# yum install leapp-data-centos
+----
```

Comment:
note to @evgeni - this package doesn't exist in the el8 repo today, but we need to do something about it